tests/testsuite/freshness.rs

```diff
@@ -502,6 +502,9 @@ fn changing_bin_features_caches_targets() {
 
             [features]
             foo = []
+
+            [profile.dev]
+            debug = false
```

Comment:
This'll fail, because the message says `[unoptimized + debuginfo]`

Reply:
Updated!